libs/adapters/tsconfig.json

```diff
@@ -4,8 +4,17 @@
     "rootDir": "src",
     "outDir": "build",
     "tsBuildInfoFile": "build/.tsbuildinfo",
-    "strict": true
+    "strict": true,
+    "paths": {
+      "@hicommonwealth/core": ["core/src"],
```

Comment:
the extended base tsconfig should take care of all lib paths:

```
"paths": {
      "@hicommonwealth/*": ["libs/*/src"]
    }
```

this is a tsc feature... why isn't pnpm recognizing this pattern?